cli/dcoscli/analytics.py

```diff
@@ -107,7 +107,7 @@ def _segment_request(path, data):
         requests.post('{}/{}'.format(SEGMENT_URL, path),
                       json=data,
                       auth=HTTPBasicAuth(key, ''),
-                      timeout=1)
```

Comment:
Can you write a small comment as to why we are doing this? Also, can we change this to use our `dcos.http` package? I remember we didn't get decent debug output (with `--log-level=...`) because we weren't using `dcos.http`.

Also, should we change `dcos.http` to do the same thing?